cfme/tests/test_appliance.py

```diff
@@ -435,3 +435,24 @@ def codename_in_stdout():
         r = appliance.ssh_client.run_command(
             r'journalctl -u evmserverd -c "{}" | egrep -i "codename: \w+$"'.format(cursor))
         return r.success
+
+
+def test_fixauth_dryrun_has_feedback(temp_appliance_preconfig):
+    """
+    Check whether the fixauth says it is running in dry mode
+
+    Polarion
+        assignee: jhenner
+        casecomponent: Appliance
+        initialEstimate: 1/60h
+
+    Testing BZ 1577303
+    """
+    appliance = temp_appliance_preconfig
+    run_command = appliance.ssh_client.run_command
+    dry_run_message = (
+        'is executing in dry-run mode, and no actual changes will be made **')
+    assert dry_run_message in run_command("fix_auth -d").output
```

Comment:
optional:
we can combine this 4 assertions but totally optional

Reply:
I don't like to combine them. I don't see a reason for doing so. I see just a reason for _not_ doing so. When they are separate, one can more easily see which one failed.